src/js/pages/jobs/JobsTable.js

```diff
@@ -42,17 +42,69 @@ class JobsTable extends React.Component {
         prop: 'name',
         render: this.renderHeadline,
         sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
+        sortFunction: this.sortJobNames,
         heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'status',
+        render: this.renderStatusColumn,
+        sortable: false,
+        heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'lastRunStatus',
+        render: this.renderLastRunStatusColumn
       }
     ];
   }
 
+  getData() {
```

Comment:
I agree with @orlandohohmeier and I also suggest this is OK for now but we should maybe revisit this approach later.